<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderLoginWidget</class>
 <widget class="QWidget" name="ReaderLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>84</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>读者登录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="m_returnButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>人脸登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_accountLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>158</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>158</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>250</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="m_loginButton">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_registerButton">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_returnButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
